src/components/VersionFileViewer/index.tsx

```diff
@@ -85,8 +86,6 @@ const VersionFileViewer = ({
             </AccordionItem>
           </AccordionMenu>
         }
-        mainSidePanelClass={styles.mainSidePanel}
```

Comment:
These styles can be removed now:

https://github.com/mozilla/addons-code-manager/blob/2b761059604b5df35c82adea6aab59dfa9ee297c/src/components/VersionFileViewer/styles.module.scss#L1-L5

(I think this change just got lost in the merge)

Reply:
Sorry for letting these lines back again....
I addressed all comments.